docs/references/javascript/user/user.mdx

```diff
@@ -58,54 +58,149 @@ The ClerkJS SDK provides some helper [methods](#methods) on the `User` object to
 
 ### `update()`
 
+Updates the user's attributes. Use this method to save information you collected about the user.
+
+The appropriate settings must be enabled in the Clerk Dashboard for the user to be able to update their attributes. For example, if you want to use the `update({ firstName })` method, you must enable the **Name** setting. It can be found in the the Clerk Dashboard under **User & Authentication > Email, Phone, Username > [Personal information](https://dashboard.clerk.com/last-active?path=user-authentication/email-phone-username).**
+
 ```typescript
 function update: (params: UpdateUserParams) => Promise<User>;
 ```
 
-Updates the user's attributes. Use this method to save information you collected about the user.
-
 #### `UpdateUserParams`
 
 All props below are optional.
 
 | Name | Type | Description |
 | --- | --- | --- |
 | `username` | `string` | The user's username. |
-| `password` (deprecated) | `string` | The user's password.<br />This property is deprecated. This will be removed in the next major version (V5). Please use [`updatePassword(params)`](/docs/references/javascript/user/password-management#update-password) instead. |
```

Comment:
This won't be fully removed till v5, but we're close enough to releasing it that I feel like we keep this change anyway. Just a note!

Reply:
oh good catch! i'm going to add it back then because i'm pretty sure we are going to have v4 vs v5 docs